src/UID2.Client/UID2Encryption.cs

```diff
@@ -271,7 +271,7 @@ internal static EncryptionDataResponse EncryptData(EncryptionDataRequest request
                     {
                         // Decryption will fail if the token is a CSTG-derived token.
```

Comment:
This comment is no longer correct as we're passing `enableDomainNameCheck: false`